scripts/check.sh

```diff
@@ -0,0 +1,5 @@
+ #'/bin/bash
+
+
+golangci-lint run /home/ashan/go/src/employee_system/
```

Comment:
No need to pass absolute path... This command will not work for other team members.

since makefile will be executed from root dir of the project.. you can simply do `golangci-lint run`